src/strategies/spidStrategy.ts

```diff
@@ -129,6 +137,39 @@ const spidStrategy = (
     }
   };
 
+  const ARUBA_ID = "https://loginspid.aruba.it";
```

Comment:
it looks like these values are used only in `IDP_IDS`, I don't see the need to create a `const` for each one of them, is it?

Reply:
fix on commit 1327add